Class3/selenium_iframes.py

```diff
@@ -9,14 +9,14 @@
 # Create a webdriver instance for chrome browser
 browser = webdriver.Chrome()
 
-browser.get('file:///home/arch-suhi/techsparks/python_ci_automation/Class3/iframe.html')                                                                                                                                              
```

Comment:
No hardcoding file paths like this. Please have the html file checked into source and refer to that as relative path

Reply:
Made required changes by refering to relative path